examples/optimisation/introduction_example/introduction_example.py

```diff
@@ -0,0 +1,56 @@
+import matplotlib.pyplot as plt
+import dhnx
+
+# Initialize thermal network
+network = dhnx.network.ThermalNetwork()
+network = network.from_csv_folder('twn_data')
+
+# Load investment parameter
+invest_opt = dhnx.input_output.load_invest_options('invest_data')
+
+# plot network
+static_map = dhnx.plotting.StaticMap(network)
+static_map.draw(background_map=False)
+plt.title('Given network')
+plt.scatter(network.components.consumers['lon'], network.components.consumers['lat'],
+            color='tab:green', label='consumers', zorder=2.5, s=50)
+plt.scatter(network.components.producers['lon'], network.components.producers['lat'],
+            color='tab:red', label='producers', zorder=2.5, s=50)
+plt.scatter(network.components.forks['lon'], network.components.forks['lat'],
+            color='tab:grey', label='forks', zorder=2.5, s=50)
```

Comment:
Not super urgent, but these lines could be integrated in StaticMap (with a default colormap and the option to pass a custom one).